libvcx/src/issuer_credential.rs

```diff
@@ -1,610 +1,23 @@
 use serde_json;
 
-use api::VcxStateType;
-use connection;
-use credential_request::CredentialRequest;
 use error::prelude::*;
-use issuer_credential_utils::encode_attributes;
-use messages;
-use messages::{GeneralMessage, MessageStatusCode, RemoteMessageType};
-use messages::get_message::get_ref_msg;
-use messages::payload::{PayloadKinds, Payloads};
-use messages::thread::Thread;
 use object_cache::ObjectCache;
-use settings;
 
-use utils::agent_info::{get_agent_attr, get_agent_info, MyAgentInfo};
-use utils::constants::CRED_MSG;
 use utils::error;
-use utils::libindy::{anoncreds, payments};
-use utils::libindy::payments::PaymentTxn;
-use utils::qualifier;
 use v3::handlers::issuance::Issuer;
 
 lazy_static! {
-    static ref ISSUER_CREDENTIAL_MAP: ObjectCache<IssuerCredentials> = ObjectCache::<IssuerCredentials>::new("issuer-credentials-cache");
+    static ref ISSUER_CREDENTIAL_MAP: ObjectCache<Issuer> = ObjectCache::<Issuer>::new("issuer-credentials-cache");
 }
 
+// This enum is left only to avoid making breaking serialization changes
```

Comment:
In fact I think in the future we might accommodate same versioning strategy as was used before, having multiple version of state machines.

Reply:
So I think it's fine if this stays